src/main/java/org/jabref/gui/contentselector/ContentSelectorDialog.java

```diff
@@ -302,8 +310,7 @@ private void applyChanges() {
 
         // Update all selectors in the current BasePanel.
         if (changedFieldSet) {
-            // We have added or removed content selectors, update the entry editor
-            panel.rebuildAllEntryEditors();
+            // TODO: We have added or removed content selectors, update the entry editor
```

Comment:
What about this TODO? This shouldn't be merged as long as the PR still breaks the feature.